packages/flutter/lib/src/rendering/wrap.dart

```diff
@@ -451,6 +453,22 @@ class RenderWrap extends RenderBox
     }
   }
 
+  /// The maximum number of lines to display.
+  ///
+  /// If null, there is no limit on the number of lines.
+  /// If not null, the wrap will stop creating new lines after reaching this limit.
+  /// Children that would exceed this limit are not displayed or laid out.
```

Comment:
This is curious behavior why would we want to ignore the children that exceed the line limit and not lay them out?

Reply:
Ah interesting! Thank you for the detailed explanation. I dd not know Text worked that way. :)